man/Dockerfile.aarch64

```diff
@@ -1,6 +1,25 @@
-FROM    aarch64/ubuntu:xenial
+FROM    aarch64/ubuntu:wily
```

Comment:
Excellent question. I did that initially so that it would be in line with the main Dockerfile.aarch64, which uses wily.